config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Defines the root path route ("/")
+  namespace :api do
+    namespace :v1 do
+      resources :roots
+      get '/message', to: 'messages#index'
+    end
+  end
+end
```

Comment:
- [ ] Please add the root to the index page to load the greetings, not to the default page (Rails logo).

`root to: 'api/v1/roots#index'`